.gitignore

```diff
@@ -389,3 +389,6 @@ fabric.properties
 
 # Custom rules (everything added below won't be overriden by 'Generate .gitignore File' if you use 'Update' option)
 
+
+.vscode/settings.json
+assets/.DS_store
```

Comment:
Make it just `.DS_Store` please, we don't want any included in the repo anywhere.

And please make sure to delete all of them from git, there are still 3 listed in the repo. Please see the files changed tab.